src/query/storages/hive/hive/src/hive_catalog.rs

```diff
@@ -368,23 +264,42 @@ impl Catalog for HiveCatalog {
     #[minitrace::trace]
     #[async_backtrace::framed]
     async fn get_database(&self, _tenant: &str, db_name: &str) -> Result<Arc<dyn Database>> {
-        let client = self.get_client()?;
-        let _tenant = _tenant.to_string();
-        let db_name = db_name.to_string();
-        tokio::task::spawn_blocking(move || Self::do_get_database(client, db_name))
+        let db = self
+            .client
+            .get_database(db_name.to_string().into())
```

Comment:
`Volo` defaults to use [`FastStr`](https://docs.rs/faststr) as its underlying String type, and you may use `FastStr::new(db_name)` here for better performance.

[`FastStr::new`](https://github.com/volo-rs/faststr/blob/ee339030b1caea5f0d080151a8cce549444db3f8/src/lib.rs#L513) is optimized for small strings that can be stored directly on stack. This can reduce malloc/free cost.